esrally/mechanic/telemetry.py

```diff
@@ -47,16 +47,12 @@ def __init__(self, enabled_devices=None, devices=None):
         self.devices = devices
 
     def instrument_candidate_env(self, car, candidate_id):
```

Comment:
I wonder whether we should rename this method because we now provide only Java options.